moses/comboreduct/atomese_representation/atomese_representation.cc

```diff
@@ -0,0 +1,148 @@
+/** atomese_representation.cc ---
+ *
+ * Copyright (C) 2018 OpenCog Foundation
+ *
+ * Author: Yidnekachew Wondimu <searchyidne@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include <moses/comboreduct/table/table_io.h>
+#include <opencog/util/comprehension.h>
+#include <opencog/atoms/base/Link.h>
+#include <opencog/atoms/base/Node.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <opencog/truthvalue/SimpleTruthValue.h>
+#include "atomese_representation.h"
+
+namespace opencog { namespace atomese {
+
+using namespace std;
+using namespace combo;
+
+Handle& create_repr(const vector<string>& labels, HandleSeq rows,
```

Comment:
It would be more efficient to pass `rows` as a const ref, like `labels`.

Reply:
Also, same remark as for `get_atomese_representation`, why not just return a `Handle` instead of taking the output Handle as ref argument?